types/test/README.md

```diff
@@ -0,0 +1,9 @@
+# Types Test
```

Comment:
I think I need another session to understand this well. Looks very promising to quickly verify for standard chains if types can be decoded 👍

Reply:
I've updated these docs, PTAL.